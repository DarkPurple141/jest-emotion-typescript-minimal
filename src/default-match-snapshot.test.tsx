import React from 'react';
import Component from './app';
import renderer from 'react-test-renderer';

/** this works but isn't as useful as desired */
test('Button renders correctly', () => {
  expect(
    renderer
      .create(<Component />)
      .toJSON()
  ).toMatchSnapshot()
})