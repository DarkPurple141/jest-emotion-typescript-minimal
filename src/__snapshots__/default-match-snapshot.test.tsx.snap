// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
.emotion-0 {
  color: red;
}

.emotion-1 {
  background-color: blue;
}

<div>
  <p
    className="emotion-0"
  >
    Hello
  </p>
  <button
    className="emotion-1"
  >
    World
  </button>
</div>
`;
